hadoop-hdds/docs/content/start/Kubernetes.md

```diff
@@ -51,3 +50,34 @@ Now you can access any of the services. By default the services are not publishe
 kubectl port-forward s3g-0 9878:9878
 kubectl port-forward scm-0 9876:9876
 ```
+
+## Apache Ozone Helm Chart
+
+For a streamlined and production-ready deployment of Apache Ozone on Kubernetes, consider using the [Apache Ozone Helm Chart](https://apache.github.io/ozone-helm-charts/). This Helm Chart simplifies the installation and management of an Ozone cluster by packaging best practices into a set of configurable Kubernetes resources.
```

Comment:
I would omit `production-ready`, implementation of HA is still in progress: https://github.com/apache/ozone-helm-charts/pull/10

Reply:
https://issues.apache.org/jira/issues/?jql=project%20%3D%20HDDS%20AND%20text%20~%20helm%20and%20status%20not%20in%20(resolved)%20ORDER%20BY%20status%20ASC

Reply:
Version 0.1.0 was the initial release (HDDS-11039).